tests/integration/synapseclient/test_wikis.py

```diff
@@ -126,3 +126,18 @@ async def test_wiki_version(syn, project):
     w2 = syn.getWiki(owner=wiki.ownerId, subpageId=wiki.id, version=1)
     assert "version 2" in w2.title
     assert "version 2" in w2.markdown
+
+
+async def test_wiki_with_empty_string_parent_wiki_id(syn, project):
+    # WHEN a wiki is created with an empty string parentWikiId
```

Comment:
A cleaner test of this would be:

GIVEN a wiki with an empty string for ParentWikiId
WHEN I store the Wiki
THEN It's stored in Synapse <-- This check should be added by performing a read to verify Synapse has the data we expect it to have
AND ...